glossary.yaml

```diff
@@ -176,8 +178,11 @@ terms:
     name: "Transaction Reciept"
   - definition: "On completion of the transaction, a transaction receipt is generated. Receiving a transaction hash does not mean that the transaction has been confirmed, it only means that the node accepted it without error, which also means that there is a high probability other producers will accept it. A transaction is considered confirmed when a nodeos instance has received, processed, and written it to a block on the blockchain, i.e. it is in the head block or an earlier block."
     name: "Confirmed Transaction"
-  - definition: "Multisig or msig is a short term for multiple signatures. It’s used to describe the case in which one requires more than one account's permission to execute a transaction. EOSIO provides the system account eosio.msig, which can be used to push onto the blockchain the multisig proposals and their corresponding account's permission required to approve the proposal. Multisig, when used properly, increases the security of an account, the security of a smart contract, and it's also the method by which Block Producers are able to affect changes within an EOSIO blockchain."
+  - definition: "Multisig is a short term for multiple signatures. It’s used to describe the case in which one requires more than one account's permission to execute a transaction. EOSIO provides the system account eosio.msig, which can be used to push onto the blockchain the multisig proposals and their corresponding account's permission required to approve the proposal. Multisig, when used properly, increases the security of an account, the security of a smart contract, and it's also the method by which Block Producers are able to affect changes within an EOSIO blockchain."
     name: Multisig
+    synonyms:
+      - msig
+      - multiple signatures
```

Comment:
the other synonyms are enclosed in double quotes, does it really make a difference?

Reply:
YAML is a bit loose. Strings technically don't always require quotes, depends on the string. That doesn't detract from style guides suggesting consistent formatting however. 

both those strings above work.

But this one would throw an error 
```
somestring: It's going to break thanks to that single quote. 
```

This would fix it 
```
somestring: "It's not going to break thanks to double quotes. "
```

Or this
```
somestring: It\'s not  going to break thanks to the escaped character
```

https://docs.octoprint.org/en/master/configuration/yaml.html#string